white-test-app-backend/src/main/java/pl/lodz/p/white/whitetestapp/accountmanager/manager/AccountManager.java

```diff
@@ -72,6 +75,6 @@ public void addRedactor(AccountDto account) throws EntityAlreadyExistsException,
     }
 
     private void createRedactorEntity(AccountDto account) throws ConstraintViolationException, IllegalArgumentException  {
-       repository.saveAndFlush(AccountMapper.fromDtoConverter(account));
+        repository.saveAndFlush(AccountMapper.fromDtoConverter(account, encoder));
```

Comment:
po co spacja

Reply:
bo wcześniej było źle sformatowane, nie wiesz kto mógł to zrobić ? :P